suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -263,7 +263,10 @@ export const acquireDevice = createThunk(
  */
 export const authorizeDevice = createThunk(
     `${MODULE_PREFIX}/authorizeDevice`,
-    async (_, { dispatch, getState, extra }): Promise<boolean> => {
+    async (
```

Comment:
Please add comment here that it's needed because of mobile and might be removed once mobile supports passphrase

Reply:
comment added : 25445f0342